tests/test_utils_cache.py

```diff
@@ -459,15 +459,15 @@ def test_scan_cache_last_modified_and_last_accessed(self) -> None:
         # Readme.md has been modified and then accessed more recently
         self.assertGreaterEqual(
             readme_file_2.blob_last_modified - readme_file_1.blob_last_modified,
-            TIME_GAP * 0.999,  # 0.999 factor because not exactly precise
+            TIME_GAP * 0.9,  # 0.9 factor because not exactly precise
         )
         self.assertGreaterEqual(
             readme_file_2.blob_last_accessed - readme_file_1.blob_last_accessed,
-            2 * TIME_GAP * 0.999,  # 0.999 factor because not exactly precise
+            2 * TIME_GAP * 0.9,  # 0.9 factor because not exactly precise
         )
         self.assertGreaterEqual(
             readme_file_2.blob_last_accessed - readme_file_2.blob_last_modified,
-            TIME_GAP * 0.999,  # 0.999 factor because not exactly precise
+            TIME_GAP * 0.9,  # 0.9 factor because not exactly precise
```

Comment:
Unrelated to this PR but it's annoying when it fails :unamused: (and I'm the one who added these tests in the first place :D)